test/JDBC/upgrade/16_11/schedule

```diff
@@ -631,3 +631,4 @@ babel_assign_nchar
 alter-function-with-weak-view
 view_sec_setting_before_17_6-or-16_10
 alter-table-null-notnull
+ascii_function_before-17_7-or-16_11
```

Comment:
As we intent to push this change in 16_11 as well (now it is 16.latest in 4X), before file should not be here. Only ```ascii_function_tests``` should be here. Also, we will need to merge th PR in 4X first, else the checks will fail here.